.github/workflows/playwright.yaml

```diff
@@ -1,22 +1,27 @@
 name: Playwright Tests
-on:
-  deployment_status:
+on: [pull_request]
+
 jobs:
-  test:
+  playwright:
+    defaults:
+      run:
+        working-directory: frontend
+
+jobs:
+  playwright:
```

Comment:
@palashgdev this is supposed to be merged with the existing playwright workflow job below.

Reply:
my bad updated 😅